src/touchhandler/TouchHandler.h

```diff
@@ -6,13 +6,25 @@ namespace Pinetime {
   namespace Controllers {
     class TouchHandler {
     public:
+      // Enum describes how the watch was woken:
+      //  * WakeUpAction: The actions selected in the wakeup settings, single/double tap, raise, shake
+      //  * Button: The hardware button
+      //  * Other: Other things that can wake the watch up, eg. apps and notifications.
+      enum class WokenBy { WakeUpAction, Button, Other };
+
+      // Enum describes how the reply from ProcessTouchInfo should be interpreted:
+      //  * NoAction: Do nothing, ignore input.
+      //  * TouchEvent: The input should be treated as a normal touch event.
+      //  * IgnoreTouchPopup: Show the popup when ignoring touch input.
+      enum class TouchProcessReply { NoAction, TouchEvent, IgnoreTouchPopup };
+
       struct TouchPoint {
         int x;
         int y;
         bool touching;
       };
 
-      bool ProcessTouchInfo(Drivers::Cst816S::TouchInfos info);
+      TouchProcessReply ProcessTouchInfo(Drivers::Cst816S::TouchInfos info, bool buttonUnlocksOn);
```

Comment:
FYI: If you set a default value for the new parameter this PR builds even without changes to InfiniSim

```suggestion
      TouchProcessReply ProcessTouchInfo(Drivers::Cst816S::TouchInfos info, bool buttonUnlocksOn=false);
```